### PR TITLE
Revise API encryption at resk task intro

### PR DESCRIPTION
**Pull request by sftim**
Split out from PR #33285

- document that you can encrypt more than Secrets
- make it clear that this _isn't_ the page for you if you want to encrypt storage volumes

/sig security
/sig auth

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/encrypt-data.md [hunk 1]
```diff
@@ -8,7 +8,26 @@ weight: 210
 ---
 
 <!-- overview -->
-This page shows how to enable and configure encryption of secret data at rest.
+
+Some of the APIs in Kubernetes, such as {{< glossary_tooltip text="Secret" term_id="secret" >}},
```
Comment (enj) [inline content/en/docs/tasks/administer-cluster/encrypt-data.md:12]: All REST APIs support encryption at rest, not just 'some'.

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/encrypt-data.md:12]: Are we sure? Maybe “Every API that can persist data?”

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/encrypt-data.md]: I reworded. How's that?

Comment (enj) [reply content/en/docs/tasks/administer-cluster/encrypt-data.md]: > Are we sure? Maybe “Every API that can persist data?”

Yeah we test and support wildcards so `*.*` will encrypt every REST resource.

File: content/en/docs/tasks/administer-cluster/encrypt-data.md [hunk 2]
```diff
@@ -8,7 +8,28 @@ weight: 210
 ---
 
 <!-- overview -->
-This page shows how to enable and configure encryption of secret data at rest.
+
+All of of the APIs in Kubernetes that let you write persistent API data support
```
Comment (enj) [inline content/en/docs/tasks/administer-cluster/encrypt-data.md:12]:
```suggestion
All of the APIs in Kubernetes that let you write persistent API resource data support
```

**Comment by reylejano** [comment]
Preview: https://deploy-preview-42181--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/encrypt-data/
This has a review from SIG Auth co-chair in this [comment](https://github.com/kubernetes/website/pull/42181#issuecomment-1649870301)
/approve

**Pull request merged**